src/ProxyManager/Exception/FileNotWritableException.php

```diff
@@ -39,24 +39,4 @@ public static function fromInvalidMoveOperation(string $fromPath, string $toPath
             $toPath
         ));
     }
-
-    public static function fromNonWritableLocation($path) : self
```

Comment:
Actually, as a better alternative, I should deprecate this method, then restore the tests and drop it in 3.0.